blog/2025-03-18-devcycle-march-hackathon.md

```diff
@@ -0,0 +1,194 @@
+---
+slug: "devcycle-march-2025-hackathon"
+title: "DevCycle March 2025 - OpenFeature Hackathon"
+date: 2025-03-18
+authors: [jonathannorris]
+description: "DevCycle March 2025 OpenFeature Hackathon"
+tags: [devcycle, hackathon]
+draft: false
+---
+
+A few times a year at DevCycle, we hit pause on our usual work, push aside our roadmaps, and dive into a few days of rapid experimentation and collaboration.
+For our first hackathon of 2025, we opted for an **OpenFeature-theme** —a chance to explore new ideas, test bold concepts, and build something innovative for the OpenFeature ecosystem together.
+<!--truncate-->
+
+This year, teams tackled projects ranging from OpenFeature CLI improvements to using OpenFeature Remote Evaluation Protocol with CloudFlare Workers.
+Let's take a closer look at how our team hacked, what we learned, and why these projects matter.
+
+## DevCycle's Hackathon Projects
+
+<img src={require('@site/static/img/blog/2025-03-18-devcycle-march-hackathon/devcycle-march-2025-hackathon.jpg').default} />
```

Comment:
Please confirm that there's nothing sensitive on the whiteboard and laptop screens. I didn't notice anything but I also didn't pull out my CSI image enhancement tools 😄

Reply:
Good catch :)